keps/sig-api-machinery/1040-priority-and-fairness/README.md

```diff
@@ -2619,6 +2619,14 @@ Beta:
   - Automatically manages versions of mandatory/suggested configuration
   - Discrimates paginated LIST requests
 
+GA:
+
+- Satisfaction with LIST and WATCH support
+- APF allows us to disable client-side rate limiting (or we know the reason why not)
+- Satisfaction that the API is sufficient to support borrowing between priority levels
```

Comment:
I would say that maybe not necessary sufficient, but we are convinced we can "extend that in backward compatible way" (i.e. we will not have to change some details fields, validations, defaulting, etc. for that purpose)

Reply:
done